### PR TITLE
(WSR 8) <Input/> - remove magnifyingGlass prop

### PR DESCRIPTION
**Pull request by EgoziE**
`<Input/>`

- Props:
  - removed `magnifyingGlass`

- Testkit:
  - removed `hasMagnifyingGlass`
  - removed `clickMagnifyingGlass`

### CODE REVIEW

File: src/Input/InputSuffix.js [hunk 1]
```diff
@@ -103,22 +97,6 @@ const InputSuffix = ({
       ),
       isVisible: suffixRules.inputHelpSuffix({ help, disabled }),
     },
-    {
-      component: () => (
-        <div
-          className={styles.magnifyingGlass}
```
Comment (simast) [inline src/Input/InputSuffix.js]: let's clean up styles as well

File: src/Input/Input.js [hunk 2]
```diff
@@ -57,11 +57,6 @@ class Input extends Component {
   };
 
   logDeprecations(props) {
-    if (props.magnifyingGlass) {
-      deprecationLog(
-        `Input's magnifyingGlass prop is deprecated and will be removed in the next major release, please use suffix property with '<Input suffix={<Input.Affix><Search /></Input.Affix>}/>' instead`,
```
Comment (simast) [inline src/Input/Input.js]: just an observation that if you try this in playground:

```jsx
<Input suffix={<Input.Affix><Icons.Search /></Input.Affix>}/>
```
the icon will not be blue by default as with old behaviour. Not sure if this is an issue?

Comment (EgoziE) [reply src/Input/Input.js]: This deprecation is in favour of `<Search/>` component.
Adding a magnifyingGlass to input is discouraged by UX (I asked Ayelet)

Comment (EgoziE) [reply src/Input/Input.js]: I'll update the migration guide

**Pull request merged**